packages/react/src/redux.ts

```diff
@@ -71,6 +77,7 @@ const ACTION_BREADCRUMB_CATEGORY = 'redux.action';
 const ACTION_BREADCRUMB_TYPE = 'info';
 
 const defaultOptions: SentryEnhancerOptions = {
+  attachReduxState: true,
```

Comment:
I'm not sure we can default this to be true because attachments have their own quota. Thoughts @HazAT?

Reply:
Alright let's :shipit: after we get some tests in @malay44!